apps/betterangels-backend/clients/enums.py

```diff
@@ -89,6 +89,7 @@ class LivingSituationEnum(models.TextChoices):
     SHELTER = "shelter", _("Shelter")
     TENT = "tent", _("Tent")
     VEHICLE = "vehicle", _("Vehicle")
+    RV = "rv", _("RV")
```

Comment:
after updating a schema or type (including the options available for a given input), you need to run `yarn nx affected -t generate-graphql-schema` to regen the schema and `yarn nx affected -t generate-graphql-types` to regen the types. alternatively `ynx-generate` will do both of the above